mantis-common/build.gradle

```diff
@@ -41,9 +40,8 @@ dependencies {
     implementation libraries.commonsIo
     implementation 'net.jcip:jcip-annotations:1.0'
 
-    testImplementation libraries.spectatorApi
+    implementation 'com.netflix.spectator:spectator-api:1.3.+'
```

Comment:
@Andyz26 I'm guessing this config change is not doable? Can you take a look?

Reply:
I don't think it's a good idea to include actual spectator lib in the common module.